docs/quickstart.md

```diff
@@ -0,0 +1,108 @@
+# Quickstart
+
+Simple-JSON was designed to be the simplest way possible for PureScript users to safely consume untyped, JSON structured data.
+
+## Reading and Writing Simple Types
+
+Simple-JSON can be used to read and write simple types, such as numbers, ints, strings, booleans, etc.
+
+For example, the snippet below attempts to parse an integer from the string `"1"` and print it to the console:
+```hs
+import Prelude
+import Data.Either (Either(..))
+import Effect (Effect)
+import Effect.Console
+import Simple.JSON as JSON
+
+main :: Effect Unit
+main =
+  case (JSON.readJSON "1") of
+    Right (int :: Int) -> 
+      log ("Parsed the integer " <> show int)
+    Left errs -> 
+      log ("Failed to parse the input as an integer: " <> show errs)
+```
+
+Note that, because `JSON.readJSON` returns `Either MultipleErrors a`, the `Right` pattern match must be explicitly annotated in this case, since there's no other way to infer that we _actually_ want to parse out an `Int` here.
```

Comment:
Technically could require a function that would then let the compiler infer that int is Int, so this should say it needs to be available in context